funnel/views/errors.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import unicode_literals
+
+from flask import current_app, redirect, request
+from werkzeug import exceptions
+from werkzeug.routing import MethodNotAllowed, NotFound, RequestRedirect
```

Comment:
You only need to import `RequestRedirect` now. The other two are available from `exceptions`